docs/guides/real-time/getting-started/websockets-graphql.md

```diff
@@ -0,0 +1,355 @@
+# Getting Started With WebSockets using GraphQL
+
+You can connect to a Directus project using a WebSocket interface and get updates on data held in a collection in
+real-time.
+
+This guide will show you how to get started with Directus' WebSockets interface, by subscribing to data using GraphQL.
+
+We'll be using a React.js project for demonstration purposes, however, WebSockets are language-agnostic, so you can
+apply the same set of steps in your stack of choice.
```

Comment:
Is there any way to do this without the use of React? I'll review expecting the answer to be "no", but if there is, I think we should do that. Less opinionated = better.